src/Consolonia.Gallery/Gallery/GalleryViews/SomeDialogWindow.axaml.cs

```diff
@@ -1,3 +1,4 @@
+#pragma warning disable CA5394 // Do not use insecure randomness
```

Comment:
next time lets supress for entire project

Reply:
I can do that.